lib/cartopy/feature/nightshade.py

```diff
@@ -29,6 +29,11 @@ def __init__(self, date=None, delta=0.1, refraction=-0.83,
         refraction : float
             The adjustment in degrees due to refraction,
             thickness of the solar disc, elevation etc...
+        sub_solar_point : tuple of latitude, longitude
```

Comment:
```suggestion
        sub_solar_point : tuple of latitude, longitude, optional
```
I guess we should note all the inputs are optional.

Reply:
I have implemented the 'optional' tag, thanks.